src/darker/import_sorting.py

```diff
@@ -33,6 +26,7 @@ def isort_code(*args: Any, **kwargs: Any) -> str:  # type: ignore[misc]
                 "An incompatible 'isort' package was found. Please install version"
                 " 5.0.0 or later."
             )
+
```

Comment:
These formatting changes most probably have nothing to do with dropping Python 3.7 support either. @hugovk was this suggested by a recent Black version, or is there some other reason?

I'd prefer to collect formatting updates in a separate PR.

Reply:
Yes, from Black.